### ISSUE TITLE
Add a progress bar

### ISSUE THREAD

**Issue by xboxones1** [nesrak1/UABEA#295]
Need a progress bar when performing various actions, when exporting/importing/saving and so on. When working with many files it is not clear whether the action has completed or not

**Comment by nesrak1** [comment]
This is coming to UABEANext.

**Comment by nesrak1** [comment]
Yes, that is why UABEANext is not fully released. I just wanted to let you know it is a planned feature in the new development version 👍 

And by the way, you can open multiple assets at one time by shift selecting them, this is one of the biggest improvements over the current version of UABEA.

**Comment by xboxones1** [comment]
I know that I can choose several files, but for me personally it is not convenient. It is necessary to look for where what is located. I liked Uabea that everything opens up to take a list and you can just filter out. Can you port this mode in UABEANEXT?

**Comment by Eepsquared** [comment]
> This is coming to UABEANext.

When? Almost 2 years later and still no progress bar when selecting multiple asset files in Workspace Explorer and having no indication (other than my HDD sound) that the files are loading. When selecting multiple asset files from Workplace Explorer to view in the center tab pane, UABEA will freeze and lag (giving no progress/status bar) indicating it is loading the selected assets. This is particularly annoying when loading, say, 291 assets (2.19GB)...

**Comment by nesrak1** [comment]
> When? Almost 2 years later and still no progress bar

I am just one person and work on this project in my free time among many other projects I work on in my free time. On top of that, there are many other features that people are asking for that seem more important since they depend on whether or not they can do something at all rather than nice UX features.

Regardless, the progress bar already exists and works when loading files for the first time. When you open a bunch of assets files or bundle files, it starts loading their headers, file listings, and names from disk. While this is happening, the progress bar at the bottom ticks away.

When you select all of the items or click the load all button, it is just putting everything together into one flat list and loading it into the data grid (which I'm not even sure is designed to be handling the millions of items I throw at it). It shouldn't even be reading from disk unless you are loading the catalog! [See here for what it's doing](https://github.com/nesrak1/UABEANext/blob/b2bd02cf8e59789a81319a292a0f9813669bf3f0/UABEANext4/ViewModels/Documents/AssetDocumentViewModel.cs#L238). I've never changed anything about the pause between clicking load all and it loading because for me, even when opening full games, this takes at max maybe 3 or 4 seconds. Are you seeing significantly longer load times?
